shoreline.go

```diff
@@ -44,8 +44,8 @@ func main() {
 
 	var mongoConfig mongo.Config
 
-	servicePort, found := os.LookupEnv("SHORELINE_PORT")
-	if !found || servicePort == "" {
+	servicePort, _ := os.LookupEnv("SHORELINE_PORT")
```

Comment:
my original remark was about using Getenv instead of LookupEnv (that only returns the value and "" if not found)

Reply:
My bad